Makefile

```diff
@@ -1,3 +1,17 @@
 DOCKER_REPO             ?= prombench
 
-include Makefile.common
\ No newline at end of file
+.PHONY: all docker docker-publish docker-manifest
```

Comment:
(nit) I prefer to have the `.PHONY` stanza defined next to the target it applies to.

```
.PHONY: all
all: ...

.PHONY: docker
docker: ...
```

Reply:
fixed